lib/web/appserver.rb

```diff
@@ -114,6 +114,12 @@ def value_to_msg(value)
   def notepad_text_path
     File.join(Narou.local_setting_dir, "notepad.txt")
   end
+
+  def get_general_all_no_by_toc(id)
+    toc = Downloader.new(id).load_toc_file
+    return nil unless toc
+    toc["subtitles"].size
```

Comment:
__[rubocop]__
* Metrics/ClassLength: Class has too many lines. [660/300]
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->